src/components/common/Extensions/ExtensionUpgradeButton/ExtensionUpgradeButton.tsx

```diff
@@ -28,19 +29,18 @@ const ExtensionUpgradeButton = ({ extensionData }: Props) => {
     [],
   );
 
-  // const isSupportedColonyVersion =
-  //   parseInt(colonyVersion || '1', 10) >= ColonyVersion.LightweightSpaceship;
-  const isSupportedColonyVersion = true;
+  if (!user?.profile || !colony) {
+    return null;
+  }
+
+  const isSupportedColonyVersion = colony.version >= 5;
 
   const extensionCompatible = isExtensionCompatible(
     Extension[extensionData.extensionId],
-    extensionData.availableVersion as ExtensionVersion,
-    10,
+    (extensionData.availableVersion + 1) as ExtensionVersion,
```

Comment:
How come we're checking if availableVersion + 1 is compatible? Isn't availableVersion the latest version in the db (i.e. latest + 1 doesn't exist)?

Reply:
Good spot, my bad!